api_incidents.go

```diff
@@ -0,0 +1,331 @@
+// Mux Go - Copyright 2019 Mux Inc.
+// NOTE: This file is auto generated. Do not edit this file manually.
+
+package muxgo
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/url"
+	"strings"
+)
+
+type IncidentsApiService service
+
+func (a *IncidentsApiService) GetIncident(iNCIDENTID string, opts ...APIOption) (IncidentResponse, error) {
+	var (
+		localVarAPIOptions   = new(APIOptions)
+		localVarHttpMethod   = strings.ToUpper("Get")
+		localVarPostBody     interface{}
+		localVarFormFileName string
+		localVarFileName     string
+		localVarFileBytes    []byte
+		localVarReturnValue  IncidentResponse
+	)
+
+	for _, opt := range opts {
+		opt(localVarAPIOptions)
+	}
+
+	// create path and map variables
+	localVarPath := a.client.cfg.basePath + "/data/v1/incidents/{INCIDENT_ID}"
+	localVarPath = strings.Replace(localVarPath, "{"+"INCIDENT_ID"+"}", fmt.Sprintf("%v", iNCIDENTID), -1)
```

Comment:
What the... I see the signature is `iNCIDENTID`, but that just feels really weird. If this is go, chalk it up to another "I'm not sure what Go is doing", but just thought I'd ask.